### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Lagicrus**
* Move some DB logic into own file to save on repeat code
* Move some lines from 1 overly long line into multi lines
* Replaced some `fetchrow` with `fetchval` or `fetch`
* Replaced 2 variables from camelCase to whatever_this_is
* Fixed some indentation
* Removed 2 variables which didn't do anything as they were written to before being read from

### CODE REVIEW

File: cogs/giftdrop.py [hunk 1]
```diff
@@ -403,18 +404,25 @@ async def change_nickname_command(self, ctx: commands.Context, target: discord.M
         if not self.bot.db_available.is_set():
             return
 
+        async with self.bot.db.acquire() as conn:
+            if not check_is_in(conn, ctx.author.id):
```
Comment (7bitlyrus) [inline cogs/giftdrop.py]: Only mods can run this command to change another user's nickname. So this should be checking the `target` not the author, and the error message reflect you're changing someone else's nickname.

File: cogs/giftdrop.py [hunk 2]
```diff
@@ -423,22 +431,22 @@ async def change_nickname_command(self, ctx: commands.Context, target: discord.M
...
                         """,
-                        target.id, 
+                        target.id,
                         nickname
                     )
                     if ret_value is None:
                         await ctx.send(f"{ctx.author.mention} Sorry, that user has not joined the event yet.")
                     else:
-                        await ctx.send(f"{ctx.author.mention}, The nickname was successfully changed to **{ret_value['nickname']}**!")
+                        await ctx.send(
```
Comment (7bitlyrus) [inline cogs/giftdrop.py]: I understand this line is a tad long, but there is no real need to split it like this for 11 characters.

File: db_stuff.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import asyncpg
```
Comment (7bitlyrus) [inline db_stuff.py]: Rename file away from `db_stuff.py` "stuff" to something more descriptive, like `db_tools.py` or `db_util.py`.

**Review by 7bitlyrus — CHANGES_REQUESTED** [review]

File: cogs/giftdrop.py (hunk 1 above)
Comment (Lagicrus) [reply cogs/giftdrop.py]: Should be dealt with now, nice catch

**Review by 7bitlyrus — APPROVED** [review]

**Pull request merged**